Makefile

```diff
@@ -474,6 +479,7 @@ include $(CONFIG_UK_BASE)/support/build/Makefile.rules
 $(foreach _M,$(wildcard $(addsuffix Makefile.rules,\
 	   $(CONFIG_UK_BASE)/arch/ $(CONFIG_UK_BASE)/arch/*/ \
 	   $(CONFIG_UK_BASE)/plat/*/ $(CONFIG_UK_BASE)/lib/*/ \
+	   $(CONFIG_UK_BASE)/drivers/*/ \
```

Comment:
TBD: Right now this allows `Makefile.rules` per sub-system (eg `drivers/ukbus/Makefile.rules`). The alternatives I see are:
* Change it to `drivers/*/*`. Allows drivers to define their own `Makefile.rules`, but makes the hierarchy of `drivers/<subsystem>/<driver>` a hard requirement.
* Implement more advanced logic that recursively search for `Makefile.rules` inside driver (not in the scope of this PR).
* Use both `drivers/*` and `drivers/*/*`. Less elegant, yet more flexible. Still somewhat limited to two-level hierarchies.

Reply:
Hum... maybe lets start with the third option and we could still improve.